hummingbird_django/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for hummingbird_django project.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.7/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.7/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'ujdo90h)l_xw$^6rj@tnmjg+^0e6zfs1cs$!vnlngjqibpg1$q'
```

Comment:
This is fine, but personally I prefer keeping the secret key out of source control. See how I've done this in other projects with `settings_secret.py`:
https://bitbucket.org/RevolutionTech/seared-quail/src/adc3cf5596c799b2b698eed01e500abd42fc4488/seared_quail/seared_quail/settings.py?at=master&fileviewer=file-view-default#settings.py-10